### PR TITLE
It support

### PR DESCRIPTION
**Pull request by Sekgomotso**
Related issues: IT Support IT Automation with Python new syllabus

## Description:

What are you up to? Fill us in :)
Created the IT Support and IT Automation with Python syllabus

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/coursera/it-automation/course-1-python-crash-course/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+_db_id:
```
Comment (ry-oc) [inline content/projects/coursera/it-automation/course-1-python-crash-course/_index.md]: remove `_db_id:` completely from every new file, tilde will add these in when it consumes the syllabus

File: content/projects/coursera/it-automation/course-1-python-crash-course/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+available_flavours:
+- none
+content_type: project
+ready: true
+submission_type: link
+title: Coursera - IT Automation with Python - Course 1 - Crash Course on Python
+---
+
+Complete the above mentioned course on coursera
```
Comment (ry-oc) [inline content/projects/coursera/it-automation/course-1-python-crash-course/_index.md]: add some submission instructions, i.e once you have completed this course take a screenshot of $whatever_you_need_to_see upload it and submit the link on the project card

File: content/projects/coursera/it-automation/course-1-python-crash-course/_index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ry-oc) [inline content/projects/coursera/it-automation/course-1-python-crash-course/_index.md]: this should be in content/coursera/... we are in the process of refactoring and wont be using the /projects folder any more

File: content/topics/time-management/_index.md [hunk 4]
```diff
@@ -57,7 +57,12 @@ Time management is an essential skill to master when tackling projects and worki
 
 ## Additional Resources to Review
 - [Time Management Skills: Definition and Examples](https://www.indeed.com/career-advice/career-development/time-management-skills)
-- [Time Management Is About More Than Life Hacks](https://hbr.org/2020/01/time-management-is-about-more-than-life-hacks)
+- [Time Management Is
+Take a look over this presentation. It'll give you the basics of CV writing. And a lot of concepts here will be the same for any kind of online portfolio you create.
```
Comment (ry-oc) [inline content/topics/time-management/_index.md]: Is this supposed to be here? Looks like it is a CV workshop

File: content/topics/time-management/_index.md [hunk 5]
```diff
@@ -57,7 +57,12 @@ Time management is an essential skill to master when tackling projects and worki
 
 ## Additional Resources to Review
 - [Time Management Skills: Definition and Examples](https://www.indeed.com/career-advice/career-development/time-management-skills)
-- [Time Management Is About More Than Life Hacks](https://hbr.org/2020/01/time-management-is-about-more-than-life-hacks)
```
Comment (ry-oc) [inline content/topics/time-management/_index.md]: did you intend to remove this link?

File: content/topics/time-management/_index.md [hunk 6]
```diff
@@ -57,7 +57,12 @@ Time management is an essential skill to master when tackling projects and worki
...
+- [Time Management Is
```
Comment (ry-oc) [inline content/topics/time-management/_index.md]: Looks like part of the link that was removed?

**Review by ry-oc — CHANGES_REQUESTED** [review]

File: content/projects/coursera/it-automation/course-2-operating-system/_index.md [hunk 7]
```diff
@@ -1,11 +1,10 @@
 ---
-_db_id:
 available_flavours:
 - none
 content_type: project
 prerequisites:
   hard:
-  - projects/coursera/it-automation/course-1-python-crash-course
+  - content/coursera/it-automation/course-1-python-crash-course
```
Comment (ry-oc) [inline content/projects/coursera/it-automation/course-2-operating-system/_index.md]: when using content links you need to leave out the `content/` so this should be `coursera/it-automation/course-1-python-crash-course`

Its also important to move the refered to content into the correct folders, this is still in the projects directory and should be int the main content directory. Apologies if my previous review was unclear on that.

**Review by ry-oc — CHANGES_REQUESTED** [review]
Almost there :)

File: content/projects/coursera/it-automation/course-2-operating-system/_index.md (hunk 7 above)
Comment (ry-oc) [reply content/projects/coursera/it-automation/course-2-operating-system/_index.md]: please let me know when you have made this change, also make sure the change is made on they syllabus

Comment (Sekgomotso) [reply content/projects/coursera/it-automation/course-2-operating-system/_index.md]: Hey,

I already did. Sorry.

**Review by ry-oc — APPROVED** [review]

**Pull request merged**